src/Todoist.Net/Models/UpdateItem.cs

```diff
@@ -0,0 +1,98 @@
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace Todoist.Net.Models
+{
+    /// <summary>
+    /// Represents a Todoist task.
+    /// </summary>
+    public class UpdateItem : BaseUnsetEntity
```

Comment:
Let's create an abstract class for Item. That will contain all the common properties for AddItem and UpdateItem to avoid duplication.

Reply:
Done.